docs/Testing.md

```diff
@@ -0,0 +1,454 @@
+# Testing with Doppl
+
+Doppl supports Android unit tests. Doppl will not only convert the production
+Java classes that you want to use on iOS, but it can convert the unit tests
+that exercise those classes. That way, you can confirm that your converted code
+works in the new environment.
+
+As with pretty much everything with Doppl, working with unit tests is mostly
+the same as what you might have done before adding Doppl to the mix. However,
+there is a bit of Doppl-specific configuration to be done, and Doppl does change
+some aspects of some of your tests.
+
+## Configuring Doppl
+
+Just as you need to teach Doppl what production classes you want to convert and
+what Java dependencies you want to use on iOS, you need to teach Doppl a bit
+about the tests that you want to run on iOS as well.
+
+### Tests and dopplConfig
+
+The `dopplConfig` closure configures Doppl. The most important configuration item
+in `dopplConfig`, for most projects, is `translatePattern`, where you indicate
+which Java classes should be converted to Objective-C. `translatePattern`
+not only needs to have your production classes, but also your test classes.
+
+In addition, Doppl offers `testIdentifier` to indicate which classes, out of
+the converted roster, are tests. By default, if you skip `testIdentifier`,
+Doppl will assume that all Java classes whose names end in `Test` are tests.
+While that default may work, it is safer to provide `testIdentifier`, lest
+you accidentally try testing something that happens to end in `Test` but
+is not a unit test.
+
+Both `translatePattern` and `testIdentifier` build up file sets in Gradle, using
+`include` and `exclude` statements:
+
+```groovy
+dopplConfig {
+    translatePattern {
+        include 'com/example/myapplication/**'
+    }
+
+    testIdentifier {
+        include 'com/example/myapplication/test/**'
+    }
+}
```

Comment:
Might be good to add an `exclude` in this example for completeness.

Reply:
I'm not sure that it's necessary here specifically, and I'm not sure that `exclude` will be all that common